resources/referers.yml

```diff
@@ -3196,12 +3196,6 @@ search:
     domains:
       - maxwebsearch.com
 
-  Meta:
```

Comment:
Maybe rename it to be more specific, rather than deleting it?

Reply:
Fair point. Unfortunately the site is in Ukrainian and the only word on it I can read is "Meta" 😀. Googling it I can find something like "Ukrainian information site Meta". Not sure what this could be renamed to.

![Screenshot 2024-10-29 at 17 28 14](https://github.com/user-attachments/assets/eb177f8e-9c38-41a6-8bc8-28b91cfe2719)

According to [Similarweb](https://www.similarweb.com/website/meta.ua/#overview) it's a pretty popular site so I'm totally open to ideas for renaming it. But since I don't have any ideas myself I'll keep this PR open.

Reply:
Maybe "Meta.ua" or "Meta (UA)"?